libs/vscode/typescript-plugin/src/lib/typescript-plugin.ts

```diff
@@ -133,14 +134,23 @@ async function getExternalFiles(
 
   for (const [, value] of Object.entries<string[]>(paths)) {
     const mainFile = join(workspaceRoot, value[0]);
+
     const configFilePath = await findConfig(mainFile, TSCONFIG_LIB);
 
     if (!configFilePath) {
       continue;
     }
 
-    const directory = dirname(configFilePath);
-    externals.push({ mainFile, directory });
+    if (mainFile.endsWith('/*')) {
+      const files = listFiles(dirname(mainFile));
```

Comment:
I don't think nested paths would work.

e.g. If `@acme/foo/*` points to `libs/foo/src`, and someone imports `@acme/foo/nested/module`.